docs/tutorials/outlook-tutorial.md

```diff
@@ -875,7 +882,7 @@ function setConfig(config, callback) {
 
 ### Create new functions to process gists
 
-Next, open the **./src/helpers/gist-api.js** file and add the following functions. Note the following:
+Next, open the *./src/helpers/gist-api.js* file and add the following functions. Note the following:
 
 - If the gist contains HTML, the add-in will insert the HTML as-is into the body of the message.
```

Comment:
Should be [as is](https://ahdictionary.com/word/search.html?q=as+is) - here and other instances
```suggestion
- If the gist contains HTML, the add-in will insert the HTML as is into the body of the message.
```